homeassistant/components/plaato/config_flow.py

```diff
@@ -1,10 +1,158 @@
-"""Config flow for GPSLogger."""
-from homeassistant.helpers import config_entry_flow
+"""Config flow for Plaato."""
+import logging
 
-from .const import DOMAIN
+from pyplaato.plaato import PlaatoDeviceType
+import voluptuous as vol
 
-config_entry_flow.register_webhook_flow(
-    DOMAIN,
-    "Webhook",
-    {"docs_url": "https://www.home-assistant.io/integrations/plaato/"},
+from homeassistant import config_entries
+from homeassistant.const import CONF_TOKEN, CONF_WEBHOOK_ID
+
+from .const import (
+    CONF_CLOUDHOOK,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_TYPE,
+    CONF_USE_WEBHOOK,
+    DOCS_URL,
+    PLACEHOLDER_DEVICE_NAME,
+    PLACEHOLDER_DEVICE_TYPE,
+    PLACEHOLDER_DOCS_URL,
+    PLACEHOLDER_WEBHOOK_URL,
 )
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__package__)
+
```

Comment:
Config flows need tests.  Please remove `plaato` from `.coveragerc` and add back only the files that do not have coverage (except `config_flow.py`).  

You can use `python3 -m script.scaffold config_flow` to help generate an example file
 pytest --cov=homeassistant/components/plaato/ --cov-report term-missing -- tests/components/plaato/test_*.py